source/data_explore.py

```diff
@@ -2,77 +2,107 @@
+#plt.show()
+
+#Feature engineering
+data_orig["gender"] = data_orig["gender"].map({"Male": 0, "Female":1})
+
+# Create Married Column - Binary Yes(1) or No(0)
+data_orig["marital-status"] = data_orig["marital-status"].replace(['Never-married','Divorced','Separated','Widowed'], 'Single')
+data_orig["marital-status"] = data_orig["marital-status"].replace(['Married-civ-spouse','Married-spouse-absent','Married-AF-spouse'], 'Married')
+data_orig["marital-status"] = data_orig["marital-status"].map({"Married":1, "Single":0})
+data_orig["marital-status"] = data_orig["marital-status"].astype(int)
+
+#Create race column - binary White-1 and nonwhite-0
+print(data_orig.race.unique())
+data_orig["race"]= data_orig["race"].replace(['Black','Asian-Pac-Islander','Other','Amer-Indian-Eskimo'],'Minorities')
+data_orig["race"]=data_orig["race"].map({"White":1,"Minorities":0})
+data_orig["race"] = data_orig["race"].astype(int)
+
+#Drop the data you don't want to use
+data_orig.drop(labels=["workclass","education","occupation","relationship","native-country"], axis = 1, inplace = True)
+print('Dataset with Dropped Labels')
+print(data_orig.head())
+print(data_orig.info())
 
 #create two datasets, male and female and create the job ids
-data_male = data_input[(data_input['gender'] == 1)]
-data_female = data_input[(data_input['gender'] == 0)]
+data_male = data_orig[(data_orig['gender'] == 1)]
+data_female = data_orig[(data_orig['gender'] == 0)]
 
 #Create test and train data based on target column TotalPay
-target_name = "TotalPay"
+target_name = "income"
+
 
+# X_train, X_test, Y_train, Y_test = create_test_train(data_orig,data_orig)
 X_train_m, X_test_m, Y_train_m, Y_test_m = create_test_train(data_male,target_name)
 X_train_f, X_test_f, Y_train_f, Y_test_f = create_test_train(data_female,target_name)
 
-acc_svc_mr_ms = svm_create_accuracy(X_train_m,Y_train_m,X_test_m,Y_test_m)
+acc_svc_mr_ms, pred = sv_accuracy(X_train_m,Y_train_m,X_test_m,Y_test_m)
 print("\n Accuracy with male train and male test data: ",round(acc_svc_mr_ms,2),"%")
+print(confusion_matrix(Y_test_m, pred))
+
 
-acc_svc_mr_fs = svm_create_accuracy(X_train_m,Y_train_m,X_test_f,Y_test_f)
+acc_svc_mr_fs, pred = sv_accuracy(X_train_m,Y_train_m,X_test_f,Y_test_f)
 print("\n Accuracy with male train and female test data: ",round(acc_svc_mr_fs,2),"%")
+print(confusion_matrix(Y_test_f, pred))
 
-acc_svc_fr_fs = svm_create_accuracy(X_train_f,Y_train_f,X_test_f,Y_test_f)
+acc_svc_fr_fs, pred = sv_accuracy(X_train_f,Y_train_f,X_test_f,Y_test_f)
 print("\n Accuracy with female train and female test data: ",round(acc_svc_fr_fs,2),"%")
+print(confusion_matrix(Y_test_f, pred))
 
-acc_svc_fr_ms = svm_create_accuracy(X_train_f,Y_train_f,X_test_m,Y_test_m)
+acc_svc_fr_ms, pred = sv_accuracy(X_train_f,Y_train_f,X_test_m,Y_test_m)
 print("\n Accuracy with female train and male test data: ",round(acc_svc_fr_ms,2),"%")
+print(confusion_matrix(Y_test_m, pred))
```

Comment:
For a future PR, it would be nice to add some functionality to print the output to a file, possibly using command line arguments.